integr8ly/installation
# INTLY-4312 ensure cluster type is set

- ensure cluster type is set
- update readme to be accurate
- remove older inventory files

verification:
- remove cluster_type from the pds group vars
- run install, ensure it fails on the prerequisite
- run upgrade, ensure it fails on the prerequisite

install logs:

```
PLAY [localhost] ********************************************************************************************************************************************************************************************************

TASK [Gathering Facts] **************************************************************************************************************************************************************************************************
ok: [127.0.0.1]

TASK [include_role : prerequisites] *************************************************************************************************************************************************************************************

TASK [prerequisites : Ensure cluster type is set] ***********************************************************************************************************************************************************************
included: /tmp/installation/roles/prerequisites/tasks/check_cluster_type.yml for 127.0.0.1

TASK [prerequisites : debug] ********************************************************************************************************************************************************************************************
fatal: [127.0.0.1]: FAILED! => {
    "msg": "cluster_type variable is not defined, please use one of the provided inventory files for your environment or set it manually"
}

PLAY RECAP **************************************************************************************************************************************************************************************************************
127.0.0.1                  : ok=2    changed=0    unreachable=0    failed=1
```

upgrade logs:

```
PLAY [master] ***********************************************************************************************************************************************************************************************************

TASK [include_role : openshift] *****************************************************************************************************************************************************************************************

TASK [openshift : Retrieve master configurations] ***********************************************************************************************************************************************************************
ok: [127.0.0.1]

TASK [openshift : add_host] *********************************************************************************************************************************************************************************************
changed: [127.0.0.1]

PLAY [master] ***********************************************************************************************************************************************************************************************************

TASK [include_role : openshift] *****************************************************************************************************************************************************************************************

TASK [openshift : Retrieve master configurations] ***********************************************************************************************************************************************************************
ok: [127.0.0.1]

TASK [openshift : add_host] *********************************************************************************************************************************************************************************************
changed: [127.0.0.1]

PLAY [localhost] ********************************************************************************************************************************************************************************************************

TASK [include_role : openshift] *****************************************************************************************************************************************************************************************

TASK [openshift : Password authentication] ******************************************************************************************************************************************************************************
skipping: [127.0.0.1]

TASK [openshift : Token authentication] *********************************************************************************************************************************************************************************
skipping: [127.0.0.1]

PLAY [localhost] ********************************************************************************************************************************************************************************************************

TASK [Gathering Facts] **************************************************************************************************************************************************************************************************
ok: [127.0.0.1]

TASK [include_role : prerequisites] *************************************************************************************************************************************************************************************

TASK [prerequisites : Ensure cluster type is set] ***********************************************************************************************************************************************************************
included: /tmp/installation/roles/prerequisites/tasks/check_cluster_type.yml for 127.0.0.1

TASK [prerequisites : debug] ********************************************************************************************************************************************************************************************
fatal: [127.0.0.1]: FAILED! => {
    "msg": "cluster_type variable is not defined, please use one of the provided inventory files for your environment or set it manually"
}
```
